olaaf_django/sync_hashes.py

```diff
@@ -61,11 +63,15 @@ def sync_hashes(repo_path):
     prev_commit = inserted_commits[-1]
 
   for commit in repo_commits[inserted_commits_num::]:
```

Comment:
Just want to confirm we are planning to replace this with the commit/hash object in the near future, correct? if so, this is good for now, if not, let's discuss.

Reply:
Yes, I created an issue for addressing that: https://github.com/openlawlibrary/OLAAF-Transient/issues/2
I updated this, but stashed the changes after we continued the discussion about the input format.